schema/patches/17.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE IF EXISTS public.terms
```

Comment:
Probably need to bump `MYRADIO_CURRENT_SCHEMA_VERSION‎` in `Controllers/root.php`